x/feerefunder/keeper/keeper.go

```diff
@@ -72,7 +73,25 @@ func (k Keeper) LockFees(ctx sdk.Context, payer sdk.AccAddress, packetID types.P
 	}
 	k.StoreFeeInfo(ctx, feeInfo)
 
-	return k.bankKeeper.SendCoinsFromAccountToModule(ctx, payer, types.ModuleName, fee.Total())
+	if err := k.bankKeeper.SendCoinsFromAccountToModule(ctx, payer, types.ModuleName, fee.Total()); err != nil {
+		return err
```

Comment:
Same thing here: please wrap errors. It very nice to have additional information in the error message!

Reply:
Yup